scripts/init_db.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e  # Stop script on error
+
+set -x #Debuging purpose 
+# Declare environment variables
+DB_USER=postgres
+DB_PASSWORD=password
+DB_DATABASE=postgres
+DB_CONTAINER_NAME=postgres_db
+DATABASE_URL="postgres://$DB_USER:$DB_PASSWORD@localhost:5432/$DB_DATABASE"
```

Comment:
Don't harcode values. Only set default values if nothing is provided.
```suggestion
DB_USER="${POSTGRES_USER:=postgres}"
DB_PASSWORD="${POSTGRES_PASSWORD:=password}"
DB_NAME="${POSTGRES_DB:=newsletter}"
DB_PORT="${POSTGRES_PORT:=5432}"
DB_HOST="${POSTGRES_HOST:=localhost}"
```